dipy/reconst/shm.py

```diff
@@ -965,3 +965,82 @@ def sh_to_sf_matrix(sphere, sh_order, basis_type=None, return_inv=True, smooth=0
         return B.T, invB.T
 
     return B.T
+
+def sh_to_ap(coeffs_mtx, normal_factor=0.00001):
+    """ Calculates anisotropic power map with a given SH coeffecient matrix
+
+    Notes
+    ----------
+    Calculate AP image based on a IxJxKxC SH coeffecient matrix based on the equation:
+    $$ AP = \sum_{l=2,4,6,...}{\frac{1}{2l+1} \sum_{m=-l}{|a_{l,m}|^2}} $$
```

Comment:
I don't think you need two `$$` here for the equation to be rendered properly. An alternative is to use the rst math directive: 

    .. math::
        AP = \sum_{l=2,4,6,...}{\frac{1}{2l+1} \sum_{m=-l}{|a_{l,m}|^2}}